mobile/bindings.go

```diff
@@ -82,8 +83,7 @@ func Start(extraArgs string, unlockerReady, rpcReady Callback) {
 	// Call the "real" main in a nested manner so the defers will properly
 	// be executed in the case of a graceful shutdown.
 	go func() {
-		if err := lnd.Main(
-			loadedConfig, cfg, signal.ShutdownChannel(),
+		if err := lnd.Main(loadedConfig, cfg, interceptor,
```

Comment:
nit: leave these params on their own newline, or flatten into a single line that ends at 80

Reply:
Thanks, I changed them back to a new line.